File: Beecrowd/Java/1114 - Fixed Password.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int X;
    while (true)
    {
        X=sc.nextInt();
        if(X==2002){
            System.out.print("Acesso Permitido\n");
            break;
        }
        else System.out.printf("Senha Invalida\n");
    }
    }
}

File: Beecrowd/C++/1114 - Fixed Password.cpp
#include<iostream>
int main(){
    int X;
    while (1)
    {
        std::cin>>X;
        if(X==2002){
            std::cout<<"Acesso Permitido\n";
            break;
        }
        else std::cout<<"Senha Invalida\n";
    }    
}
